src/main/perl/ComponentProxy.pm

```diff
@@ -336,7 +337,7 @@ sub _setDependencies {
             push (@{$self->{'PRE_DEPS'}},$el->getStringValue());
         }
         $self->debug(2, "pre dependencies for component $self->{'NAME'}: ",
-                     ' '.join(',',@{$self->{PRE_DEPS}}));
+                     join(',',@{$self->{PRE_DEPS}}));
```

Comment:
Is this formatting change really needed. It may breaks some scripts parsing this line. Looks gratuitous for me, doesn't really bring any value as a component name cannot contain a space.